tools/stern/src/Stern/Intra.hs

```diff
@@ -996,12 +996,13 @@ getOAuthClient :: OAuthClientId -> Handler OAuthClient
 getOAuthClient cid = do
   b <- view brig
   r <-
-    rpc'
-      "brig"
-      b
-      ( method GET
-          . Bilge.paths ["i", "oauth", "clients", toByteString' cid]
-      )
+    lift $
+      rpc'
+        "brig"
+        b
+        ( method GET
+            . Bilge.paths ["i", "oauth", "clients", toByteString' cid]
+        )
```

Comment:
this changes semantics but I don't think it's problematic as this is a direct call to `rpc'` which is what we do normally outside of `ExceptT` as well.